.gitignore

```diff
@@ -8,4 +8,3 @@ wasi-testsuite/
 dependency-reduced-pom.xml
 jmh-result.json
 scripts/jmh-tmp
-main/
```

Comment:
I agree the naming is not ideal but this is the folder where the baseline chicory repository gets cloned for jmh tests, I would keep it in `gitignore`

Reply:
it should be `/main/` and is fixed in the `main` branch already.